src/translib/translib.go

```diff
@@ -148,12 +138,8 @@ func Create(req SetRequest) (SetResponse, error){
 	}
 
 	writeMutex.Lock()
-	d, err := db.NewDB(db.Options {
-                    DBNo              : db.ConfigDB,
-                    InitIndicator     : "CONFIG_DB_INITIALIZED",
-                    TableNameSeparator: "|",
-                    KeySeparator      : "|",
-                      })
+
+	d, err := db.NewDB(getDBOptions(db.ConfigDB))
```

Comment:
This change needs to be checked properly. I have discarded dell team's change and taken Anan's change hoping codes were doing the same job.